azure/windows-10-agent.amd64.json

```diff
@@ -0,0 +1,57 @@
+{
+  "variables": {
+    "maven_version": "3.6.3",
+    "git_version": "2.25.1",
+    "jdk11_version": "11.0.6+10",
+    "jdk8_version": "8u242-b08",
```

Comment:
Update to latest?

Reply:
I was keeping it the same as the other Windows packer images for now, we can update all of the images to latest in another PR. Is that ok with you?

Reply:
It sounds good to me, and I am also in favor to use the same version everywhere